spec/lib/config_spec.rb

```diff
@@ -38,6 +38,19 @@ module Fusuma
       }
     end
 
+    let(:keymap_with_trigger_threshold) do
+      {
+        'swipe' => {
+          3 => {
+            'left' => {
+              'shortcut' => 'alt+Left',
+              'threshold' => 0.8
+            },
```

Comment:
__[RuboCop]__

Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.


<!-- Comment by [Sider](https://sider.review) -->